homeassistant/components/met_eireann/config_flow.py

```diff
@@ -0,0 +1,77 @@
+"""Config flow to configure Met component."""
+from typing import Any, Dict, Optional
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_ELEVATION, CONF_LATITUDE, CONF_LONGITUDE, CONF_NAME
+from homeassistant.core import callback
+import homeassistant.helpers.config_validation as cv
+
+from .const import DOMAIN, HOME_LOCATION_NAME
+
+
+@callback
+def configured_instances(hass):
+    """Return a set of configured SimpliSafe instances."""
+    entries = []
+    for entry in hass.config_entries.async_entries(DOMAIN):
+        entries.append(
+            f"{entry.data.get(CONF_LATITUDE)}-{entry.data.get(CONF_LONGITUDE)}"
+        )
+    return set(entries)
+
+
+class MetEireannFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
+    """Config flow for Met Eireann component."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+
+    def __init__(self):
+        """Init MetEireannFlowHandler."""
+        self._errors = {}
+
+    async def async_step_user(self, user_input=None):
+        """Handle a flow initialized by the user."""
+        self._errors = {}
+
+        if user_input is not None:
+            if (
+                f"{user_input.get(CONF_LATITUDE)}-{user_input.get(CONF_LONGITUDE)}"
+                not in configured_instances(self.hass)
+            ):
+                return self.async_create_entry(
+                    title=user_input[CONF_NAME], data=user_input
+                )
+            self._errors[CONF_NAME] = "name_exists"
+
+        return await self._show_config_form(
+            name=HOME_LOCATION_NAME,
+            latitude=self.hass.config.latitude,
+            longitude=self.hass.config.longitude,
+            elevation=self.hass.config.elevation,
```

Comment:
The default elevation from hass is in meters irregardless of the choice of unit system.  See issue: #34781.  I'm not sure how to advise you for this integration, but as written, this integration will have the same issue.

Reply:
Thanks for all of your suggestions. I may be misunderstanding this issue but as I understand it the elevation measurement is in metres no matter what so as long as it's not being converted to feet somewhere it should be okay?

Reply:
Well since no one responded to the issue, I'm not sure how to advise.  My understanding of the situation is that `hass.config.elevation` units are currently in meters, so if you take out the conversion to feet in the code it should be fine.  But, since this has not been confirmed one way or the other, I would wait for a second opinion.